proposals/0029-cooperative-vector.md

```diff
@@ -602,92 +615,90 @@ The vector-matrix intrinsics are expected to be supported in all shader stages.
 
 #### D3D12 API Additions
 
-Note: The enums and structs need to be updated from the coop_vec name, once a
-new name for the feature is decided.
-
 ### Check Feature Support
 
 This feature requires calling CheckFeatureSupport(). Additional D3D12_FEATURE
 enum and corresponding D3D12_FEATURE_DATA* structs (listed below) are added to
-enable discovering the Cooperative Vector tier along with the datatype and
+enable discovering the Linear Algebra: Matrix Vector tier along with the datatype and
```

Comment:
Is `Linear Algebra: Matrix` a namespaced enum or something? The spaces are confusing me and I don't think the colon is serving a grammatical function.

Reply:
I was attempting to mash implied namespaces into prose.  Would Linear Algebra Matrix Vector be fine?

Reply:
Or just "Linear Algebra Vector"? 🤔 (matrix vector sounds weird)

Reply:
for the tier went back to cooperative vector naming.